tests/pages/services/ServiceActions-cy.js

```diff
@@ -51,16 +51,6 @@ describe('Service Actions', function () {
 
   context('Destroy Action', function () {
 
-    context('Framework', function () {
```

Comment:
This was removed because the behavior it's testing is not desired any longer. Some time ago, we weren't providing the `Destroy` option for Frameworks, but now we are. Here's the relevant JIRA task: https://mesosphere.atlassian.net/browse/DCOS-12269